internal/common/scheme.go

```diff
@@ -19,5 +19,5 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(Scheme))
 	utilruntime.Must(extv1.AddToScheme(Scheme))
 	utilruntime.Must(sspv1beta1.AddToScheme(Scheme))
-	utilruntime.Must(osconfv1.AddToScheme(Scheme))
+	utilruntime.Must(osconfv1.Install(Scheme))
```

Comment:
What does this change do?

Reply:
`AddToScheme` is deprecated, `Install` is the same function, with different name:
https://github.com/openshift/api/blob/master/config/v1/register.go#L21

Reply:
So can you please change it for all schemas?

Reply:
Not all modules rename it. Some keep the `AddToScheme` name. I guess they will rename it in later versions.

Reply:
You are right, it is only deprecated in OpenShift API.